dbms/src/Columns/ColumnArray.h

```diff
@@ -172,10 +209,7 @@ class ColumnArray final : public COWPtrHelper<IColumn, ColumnArray>
 
     std::pair<UInt32, StringRef> getElementRef(size_t element_idx) const;
 
-    size_t ALWAYS_INLINE sizeAt(size_t i) const
-    {
-        return i == 0 ? getOffsets()[0] : (getOffsets()[i] - getOffsets()[i - 1]);
-    }
+    size_t ALWAYS_INLINE sizeAt(ssize_t i) const { return (getOffsets()[i] - getOffsets()[i - 1]); }
```

Comment:
Does we ensure `getOffsets[-1] == 0` is well-defined?

Reply:
Yes, you can see the comments here https://github.com/pingcap/tiflash/blob/5dd3a733a2dec9ee3548353356583b21cd03d54d/dbms/src/Common/PODArray.h#L68C2-L69C93
and code here https://github.com/pingcap/tiflash/blob/5dd3a733a2dec9ee3548353356583b21cd03d54d/dbms/src/Common/PODArray.h#L68C2-L69C93.